standard/types.md

```diff
@@ -537,11 +537,15 @@ type_arguments
     ;   
 
 type_argument
-    : type
+    : type nullable_argument_attribute?
+    ;
+
+nullable_argument_attribute
```

Comment:
Note for Nigel: This enables nullable annotations on a type argument. It follows the structure you suggested in types for nullable annotation on type parameters. 

It removes the `??` from the grammar, but also does repeat the same syntax as different productions. Suggestions welcome.

Reply:
@BillWagner – The attribute applies to types and we don't need two rules. Use *nullable_type_attribute* here. At the first use in the Standard (probably here?) define it and just use it elsewhere (so probably remove the definition in classes.md).